rnndb/fifo/gk104_copy.xml

```diff
@@ -36,14 +36,8 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 				<value value="1" name="BLOCK"/>
 				<value value="2" name="NONBLOCK"/>
 			</bitfield>
-			<bitfield low="7" high="7" name="SRC_MODE">
-				<value value="0" name="2D_LINEAR"/>
-				<value value="1" name="3D_TILED"/>
-			</bitfield>
-			<bitfield low="8" high="8" name="DST_MODE">
-				<value value="0" name="2D_LINEAR"/>
-				<value value="1" name="3D_TILED"/>
-			</bitfield>
+			<bitfield pos="7" name="SRC_FORMAT" type="memory_format"/>
```

Comment:
That should not be called FORMAT - that collides with, well, the surface format (as in, A8R8G8B8 etc.)
Perhaps MODE?